Data_storage/DataStore.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+
+class DataStorage:
+    def __init__(self):
```

Comment:
it is the same problem .. connect with Iliya about `__init__` - he can tell you what s wrong here